tests/integration/hooks.bats

```diff
@@ -42,3 +42,41 @@ function teardown() {
 		[[ "$output" == *"error running $hook hook #1:"* ]]
 	done
 }
+
+@test "runc run [hook with env]" {
+	update_config '.process.args = ["/bin/true"]'
+	update_config '.process.env = ["mm=nn"]'
```

Comment:
What if we use a more readable name?
```suggestion
	update_config '.process.env = ["container_var=yes"]'
```

Reply:
Because we do a test in one loop `for hook in prestart createRuntime createContainer startContainer poststart; do`, but as mentioned in https://github.com/opencontainers/runc/pull/4323#discussion_r1648764306, the var may be in host, or may be in container, but we should use one var name, so we should not specify the var name.

Reply:
What about self-descriptive names such as `TEST_VAR` for variable name, and `host_value` and `TEST_VAR=container_value` (for values), or some such, to increase the test code readability.